charts/tidb-operator/templates/scheduler-deployment.yaml

```diff
@@ -48,7 +48,7 @@ spec:
           value: {{ .Values.timezone | default "UTC" }}
       {{- end }}
       - name: kube-scheduler
-        image: {{ required "scheduler.kubeSchedulerImageName is required" .Values.scheduler.kubeSchedulerImageName }}:{{ .Values.scheduler.kubeSchedulerImageTag | default (split "-" .Capabilities.KubeVersion.GitVersion)._0 }}
+        image: {{ required "scheduler.kubeSchedulerImageName is required" .Values.scheduler.kubeSchedulerImageName }}:{{- template "kube-scheduler.image_tag" . }}
```

Comment:
the old way only works for `vX.Y.Z-xxx` version, however the version of OpenShift is `v1.17.1+3f6f40d`